tools/nodejs/src/database.cpp

```diff
@@ -28,9 +28,11 @@ struct OpenTask : public Task {
 
 	void DoWork() override {
 		try {
-			Get<Database>().database = duckdb::make_unique<duckdb::DuckDB>(filename);
-			duckdb::ParquetExtension extension;
-			extension.Load(*Get<Database>().database);
+			duckdb::DBConfig config;
+			config.maximum_threads = 1;
+			Get<Database>().database = duckdb::make_unique<duckdb::DuckDB>(filename, &config);
+			//			duckdb::ParquetExtension extension;
```

Comment:
Should parquet extension remain disabled here?

Reply:
Good catch that was accidental